english/bird-salvation/bird.md

```diff
@@ -4,3 +4,7 @@ marshmallow right off of you.
 	Amazed, you sit in silence at the mythical animal you just witnessed ...
 as well as trying to catch your breath...and, in the distance, glimpse
 what appears to be a horse with a horn sprouting from its forehead...
+
+You really have only one option
+[Go ask Alice](..ask-to-alice/ask-to-alice.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.